apps/passport-server/src/util/telegramHelpers.ts

```diff
@@ -246,9 +246,9 @@ const generateProofUrl = (
     typeof ZKEdDSAEventTicketPCDPackage
   >(passportOrigin, returnUrl, ZKEdDSAEventTicketPCDPackage.name, args, {
     genericProveScreen: true,
-    title: "",
+    title: "ZK TG Auth",
```

Comment:
suggestion: I'm not sure TG or Auth are super clear as acronyms to a general audience. I suggest something like "ZK Telegram Proof"

Reply:
@rrrliu Brian suggested this as the title, but said it could be subject to change. One thing to note is that with longer titles, the overflow looks kinda bad.

<img width="383" alt="CleanShot 2023-10-11 at 15 43 07@2x" src="https://github.com/proofcarryingdata/zupass/assets/42984734/6a123094-2f73-4b2f-81c7-788284bdee32">

Relevant for your comment below imo.

Reply:
Exactly. That's why I felt leaving it empty might look better if we can't convey enough useful information. The other option is to reduce the size from H2 to H3/4